frontend/public/locales/en/translation.json

```diff
@@ -2416,6 +2416,7 @@
     "The name of the vSphere cluster to use.": "The name of the vSphere cluster to use.",
     "The name of the vSphere datacenter to use.": "The name of the vSphere datacenter to use.",
     "The OpenStack clouds.yaml file, including the password, to connect to the OpenStack server.": "The OpenStack clouds.yaml file, including the password, to connect to the OpenStack server.",
+    "The OpenStack instance type of the gateway node that will be created on the managed cluster (default PnTAE.CPU_16_Memory_32768_Disk_80).": "The OpenStack instance type of the gateway node that will be created on the managed cluster (default PnTAE.CPU_16_Memory_32768_Disk_80).",
```

Comment:
When we run cloud prepare using subctl, the default instanceType is PnTAE.CPU_4_Memory_8192_Disk_50, which does not match with this change. Lets use consistent values across subctl and ACM.

https://github.com/submariner-io/subctl/blob/95a3edd76b3e75b35453941021e10ab468b80b37/cmd/subctl/rhos.go#L78

Reply:
done.